controllers/result.go

```diff
@@ -77,6 +80,52 @@ func (c *ResultController) Create() {
 	c.Redirect("/", http.StatusFound)
 }
 
+func (c *ResultController) Show() {
```

Comment:
Based on the receiver name convention, the receiver name should be `r` or `rc`

Reply:
Fixed on [3e5e714](https://github.com/carryall/go-google-scraper-challenge/pull/72/commits/3e5e714ac9ac92748a798395d061496ae05ba81a)